cyto_dl/models/vae/image_canon_vae.py

```diff
@@ -124,7 +124,7 @@ def __init__(
                 )
             )
 
-        if isinstance(prior, (str, type(None))):
+        if prior is None or isinstance(prior, None):
```

Comment:
this feels like its not checking for prior being a str?

Reply:
yikes good catch

Reply:
fixed!